src/routes/guild/create/index.ts

```diff
@@ -16,17 +15,10 @@ export class UserRoute extends Route {
 	public async [methods.POST](request: ApiRequest<GuildCreateQuery>, response: ApiResponse) {
 		const { guild_id, inviter } = request.query;
 
-		try {
-			const [_createGuild] = await container.sequelize.query('INSERT INTO guild (guild_id, inviter) VALUES (?,?)', {
-				replacements: [guild_id, inviter],
-			});
-		} catch (error: any) {
-			if (error.name === 'SequelizeUniqueConstraintError') {
-				return response.badRequest({ message: 'Guild already exists', guild_id, inviter: inviter });
-			}
-			DEBUG ? container.logger.error(error) : null;
-			return response.error(error);
-		}
-		return response.ok({ message: `Guild ${guild_id} created` });
+		const guild = await container.utilities.guild.create({ guild_id, inviter });
+
+		if (!guild) return response.badRequest({ error: 'Guild not found' });
```

Comment:
Shouldn't this be a different Error message? Something like `something went wrong while creating the guild`?